openmicroscopy/openmicroscopy
# Insight offline

# What this PR does

Bring to the mainline the customization made to the client so it can work with omero-smuggler

This PR depends on https://github.com/openmicroscopy/omero-ms-queue/pull/18


# Testing this PR
* You will need to deploy locally omero-smuggler. See https://github.com/openmicroscopy/omero-ms-queue/pull/18
* the user used for testing needs to have an email address set
* in container.xml replace
``
<entry name="omero.client.import.offline.enabled" type="boolean">false</entry>
``
by
``
<entry name="omero.client.import.offline.enabled" type="boolean">true</entry>
``
* Import an image
* Check that it imports correctly

# Related reading

https://trello.com/c/sgiFlGNk/27-integrate-offline-import-to-mainline

cc @c0c0n3 @dominikl
